packages/database/src/modelClasses/SurveyResponse.js

```diff
@@ -19,6 +19,7 @@ const USERS_EXCLUDED_FROM_LEADER_BOARD = [
   "'josh@sussol.net'", // mSupply API Client
   "'unicef.laos.edu@gmail.com'", // Laos Schools Data Collector
   "'tamanu-server@tupaia.org'", // Tamanu Server
+  "'public@tupaia.org'", // Public User
```

Comment:
I wonder if we should have some sort of common blacklist to refer to, since this seems like it's in multiple places